src/game.py

```diff
@@ -445,7 +451,7 @@ async def main():
         deal_str, auction_str = next(deal_source)
         driver.set_deal(deal_str, auction_str)
 
-        driver.human = [False, False, False, False]
+        driver.human = [0.1, 0.1, 0.1, 0.1]
```

Comment:
what was the motivation for changing this from booleans to floats?

Reply:
0.1 is the trust factor the bot is using when making search. So you can try different trust-factors
1 is human and -1 is no search
This allows replay of a given deal with changed parameters without having to change configuration or restart the server.

Reply:
ok, i see. 
this was showing which players are human and which players are robots. that's why i was confused :)
now it's showing the trust score as well as who is human.
i think it would improve clarity to have separate parameters, one showing who is human/bot, and the other giving the trust parameter

Reply:
Agree, I will change that. In principle it is a configuration, that should be loaded for each pair, as Search is just one of the interesting thresholds